clang/test/Analysis/builtin_overflow_notes.c

```diff
@@ -0,0 +1,30 @@
+// RUN: %clang_analyze_cc1 -analyzer-checker=core -analyzer-output text \
+// RUN:   -verify %s
+
+void test_no_overflow_note(int a, int b)
+{
+   int res;
+
+   if (__builtin_add_overflow(a, b, &res)) // expected-note {{Assuming overflow does not happen}}
```

Comment:
Maybe it's just me, but I find this phrasing a bit weird.
To me it would read a lot more natural if it was `Assuming overflow` or `Assuming no overflow`. Maybe suffixed with `... happens`.

Reply:
I agree that the phrasing suggested by @steakhal would be better, but I think that the original is also acceptable.